perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -92,6 +100,14 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 	<bean id="creationEventProcessor" class="cz.metacentrum.perun.ldapc.processor.impl.CreationEventProcessor">
 		<property name="dispatchConditions">
 			<list>
+				<bean class="cz.metacentrum.perun.ldapc.processor.impl.RegexpDispatchEventCondition">
+					<property name="beansCondition">
+						<list>
+							<value>cz.metacentrum.perun.core.api.Facility</value>
+						</list>
+					</property>
+					<property name="pattern" value=" created.$" />
```

Comment:
We have again wrong message in AuditEvent. Keep the regex here and update produced message``FacilityCreated`` class in project ``perun-base``, package ``cz.metacentrum.perun.audit.events.FacilityManagerEvents``. Thank you.

Reply:
Updated